modules/bbp-core/src/main/resources/schemas/bbp/core/brainLocation/v0.1.0.json

```diff
@@ -0,0 +1,117 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpvect3dsh": "{{base}}/schemas/bbp/core/vector3d/v0.1.0/shapes/",
+      "bbptlotsh": "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/core/brainlocation/v0.1.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/bbp/core/vector3d/v0.1.0"
+  ],
+  "@type": "owl:Ontology",
+  "shapes": [
+    {
+      "@id": "this:BrainLocationShape",
+      "@type": "sh:NodeShape",
+      "label": "Brain location shape definition",
+      "property": [
+        {
+          "path": "nsg:brainRegion",
+          "name": "Brain region",
+          "description": "Location as brain region",
+          "node": "this:BrainRegionShape",
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:brainLocationCoordinates",
+          "name": "Brain location coordinates",
+          "description": "Brain location coordinates",
+          "node": "this:BrainLocationCoordinatesShape",
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:sliceLocationCoordinates",
+          "name": "Slice location coordinates",
+          "description": "Slice location coordinates",
+          "node": "this:SliceLocationCoordinatesShape",
+          "maxCount": 1
+        }
+      ]
+    },
+    {
+      "@id": "this:BrainRegionShape",
+      "@type": "sh:NodeShape",
+      "label": "Brain region shape definition",
+      "property": [
+        {
+          "path": "nsg:brainRegion",
+          "name": "Brain region",
+          "description": "Brain region",
+          "node": "bbptlotsh:BrainRegionOntologyTermShape",
+          "minCount": 1
+        }
+      ]
+    },
+    {
+      "@id": "this:BrainLocationCoordinatesShape",
+      "@type": "sh:NodeShape",
+      "label": "Brain coordinates shape definition",
+      "and": [
+        {
+          "node": "bbpvect3dsh:Vector3DShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:spatialReferenceSystem",
+              "name": "Spatial Reference System",
+              "class": "nsg:SpatialReferenceSystem",
+              "minCount": 1,
+              "maxCount": 1
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "this:SliceLocationCoordinatesShape",
```

Comment:
Shouldn't this shape be defined just like the the brain location coordinates ?

Reply:
There are a few differences in the coordinates given in slice and the coordinates given in the brain. Slice location coordinates ...
... can be 2 dimensional
... don't always come with a SpatialReferenceSystem (as sometimes the coordinates are taken with respect to an arbitrary origin set by the microscope)

Reply:
Okay. I see. 
I think the SpatialReferenceSystem should always be present. In case of a location in a slice, the SpatialReferenceSystem can correspond to the slice itself. Is it correct @huanxiang .